app/admin/subscriptions.rb

```diff
@@ -14,6 +14,24 @@
     actions
   end
 
+  show do
+    attributes_table do
+      row :id
+      row :kind
+      row :expires_at
+      row :trial_ended_at
+      row :activated_at
+      row :chargify_subscription_id do |subscription|
+        if subscription.chargify_subscription_id
+          link_to subscription.chargify_subscription_id,
+                  ChargifyService.new(subscription.chargify_subscription_id).subscription_url, target: '_blank'
```

Comment:
I don't think I quite see the merit of including ChargifyService in this. Options:

- Create a helper method, `chargify_subscription_url(subscription)`
- Put a `chargify_url` method on subscription.rb, `subscription.chargify_url`
- Create a SubscriptionDecorator which wraps a subscription `subscription.decorate.chargify_url` (most correct, but we don't do this anywhere else yet.)

I'd reckon #1 would be best for now.